apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Playground/Chat.tsx

```diff
@@ -306,10 +308,31 @@ export function TokenUsage({
       )}
     >
       {responseStream === undefined ? (
-        <Text.H6M color='foregroundMuted'>
-          Prompt tokens: {usage?.promptTokens} | Completion tokens:{' '}
-          {usage?.completionTokens}
-        </Text.H6M>
+        <div className='flex flex-row gap-2'>
+          <Text.H6M color='foregroundMuted'>
+            {usage?.totalTokens} tokens
+          </Text.H6M>
+          <Tooltip
+            side='right'
```

Comment:
Put it on `'top'`

Reply:
I did ! You are pointing the previous commit :)